acmeissuer.go

```diff
@@ -223,7 +230,7 @@ func NewACMEIssuer(cfg *Config, template ACMEIssuer) *ACMEIssuer {
 		}
 	}
 	transport := &http.Transport{
-		Proxy:                 http.ProxyFromEnvironment,
+		Proxy:                 template.HTTPProxy,
```

Comment:
We can apply the default of `http.ProxyFromEnvironment` (around) here instead, if `template.HTTPProxy` is nil.